src/stylesheets/elements/_buttons.scss

```diff
@@ -155,18 +155,53 @@ button,
 .usa-button:disabled,
 .usa-button-disabled {
   background-color: $color-gray-lighter;
-  color: $color-gray-dark;
   pointer-events: none;
 
   &:hover,
   &.usa-button-hover,
   &:active,
   &.usa-button-active,
-  &:focus {
+  &:focus,
+  &.usa-button-focus {
     background-color: $color-gray-lighter;
     border: 0;
     box-shadow: none;
-    color: $color-gray-dark;
+  }
+}
+
+.usa-button-secondary:disabled,
+.usa-button-secondary-disabled {
```

Comment:
Based on my understanding of the WDS CSS, I'd expect there not to be another `usa-button-secondary-disabled` class, but that if you add the `usa-button-disabled` to a button with `usa-button-secondary` it would correctly apply the right disabled styles. I definitely think `.usa-button-secondary:disabled` makes sense though